packages/@n8n/chat/package.json

```diff
@@ -5,7 +5,7 @@
     "dev": "pnpm run storybook",
     "build": "pnpm build:vite && pnpm build:bundle",
     "build:vite": "vite build",
-    "build:bundle": "INCLUDE_VUE=true vite build",
+    "build:bundle": "cross-env INCLUDE_VUE=true vite build",
```

Comment:
this line is the only change that should be in this PR to be approved. can you please revert all other changes?